### PR TITLE
fix: silence gnu parallel

### PR DESCRIPTION
**Pull request by dpc**
GNU Parallel displays this annoying prompt to give them money.

Make it go away for everyone automatically.

### CODE REVIEW

File: misc/git-hooks/pre-commit [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 set -eo pipefail
 
+# https://stackoverflow.com/a/72183258/134409
+yes 'will cite' | parallel --citation 2>/dev/null 1>/dev/null || true
```
Comment (elsirion) [inline misc/git-hooks/pre-commit:6]: :man_facepalming:

**Review by elsirion — APPROVED** [review]

**Comment by dpc** [comment]
Great. `yes 'will cite' | parallel --citation 2>/dev/null 1>/dev/null || true` hangs in CI. Probably does something different without tty.

**Comment by elsirion** [comment]
That's the kind of stuff why I never use `parallel` :grimacing: it's one of the most annoying pieces of OS software. But it's cleaner than a loop and detaching processes, so let's keep it.

**Comment by dpc** [comment]
Next time I'll try https://gitlab.redox-os.org/redox-os/parallel

**Pull request merged**